new_core/fps.js

```diff
@@ -0,0 +1,41 @@
+/* fps.js
+ *
+ * Class for quickly getting the frames per second
+ */
+
+const time = {
+
+  jfps: 0,
+  tfps: 0,
+  count: 0,
+  frame: 0,
+
+  nowTime: 0,
+  lastTime: 0,
+  diffTime: 0
+}
+
+// default lx=4, ly=10
+export function display(ctx, lx, ly) {
+  time.nowTime = new Date().getTime();
+  time.diffTime = time.nowTime - time.lastTime;
+  time.tfps = parseInt(1000 / (time.diffTime || 1), 10);
```

Comment:
I still don't see the advantage of this one. Why don't extend the performance module by fps and tfps ? 

Like you can ask performance for the fps of the whole game loop and render it by the screen module. then only screen knows about rendering and performance about time calculation. this module knows both (or better is coupled to both terms)

this is something we should discuss at least. :)

Reply:
we solving this in a new issue